### PR TITLE
Feat (index.html): don't use Mathjax during spec-generation

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
This is blocking the ReSpec from ditching Phantomjs and using ES6 features: https://github.com/w3c/respec/pull/678

Would be great to have your feedback so we can coordinate on a smooth transition.

**Comment by marcoscaceres** [comment]
I see that https://github.com/htacg/tidy-html5/issues/273 has been fixed, so the above CI issue is no longer valid?

**Comment by marcoscaceres** [comment]
Gentle ping. This is blocking ReSpec. Let me know if you need more time, but I'd like to merge the ReSpec PR later this week. It will break spec generation for this spec, but you will be able to fix it with this PR.

File: index.html [hunk 1]
```diff
@@ -95,15 +92,47 @@
...
-    // This allow wrapping mathjax formulas in <pre> so the LaTeX code is not
-    // auto wrapped and keeps a nice formatting.
-    if (window.MathJax !== undefined) {
-      MathJax.Hub.Config({
-        tex2jax: {
-           skipTags: ["script","noscript","style","textarea","code"]
-        }
+    window.addEventListener("DOMContentLoaded", function () {
+      "use strict";
+      // don't run MathJax during spec generation
```
Comment (rtoy) [inline index.html]: "dont" -> "Don't"

**Comment by rtoy** [comment]
lgtm.

@padenot WDYT?

**Pull request merged**